text/0000-issue-workflow.md

```diff
@@ -0,0 +1,222 @@
+```
+
+
+### Actionable Issues
+
+An actionable issue is one that can be worked on such that the following is true:
+
+- Definition of done - that may be acceptance criteria or clear expectations.
+- Appropriate labels
+    - `type/*`
+    - `size/*` (if possible)
+    - `status/ready`
+    - `good first issue` (if applicable)
+
+
+### Triage
+
+The following is a flow assist contributors in assigning the appropriate labels in script format.
+
+#### Issues
+
+- Is this a support request?
+    - Yes
+        - Add label type/support
+        - Provide support and close the issue with label `status/resolved`\
+          OR 
+        - Request further information from the issue author and label `status/incomplete`
+        - If activity is missing for 7 business days, follow up or close
+    - No
+        - See below
+- Is this something we want to do?
+    - I don’t even understand this issue
+        - Remove label `status/triage`
+        - Add label `status/invalid`
+        - Comment and close
+    - No - not valid / relevant for our project
+        - Remove label status/triage
+        - Add label `status/not-planned` or `status/invalid`
+        - Comment and close
+    - Maybe - needs RFC
+        - Remove label `status/triage`
+        - Add label `status/requires-rfc`
+        - Comment and close
+    - Maybe - needs alignment
+        - Remove label `status/triage`
+        - Add label `status/discussion-needed`
+        - Add appropriate `type/*`
+    - Maybe - I can’t tell from the information in the issue
+        - Remove label `status/triage`
+        - Add label `status/incomplete`
+        - Request further information from issue author
+        - If activity is missing for 7 business days, follow up or close
+    - Yes - but it’s not on our roadmap (would consider a PR)
+        - Remove label `status/triage`
+        - Add label `status/not-planned`
+        - Comment and close
+    - Yes
+        - Could this be worked on right now?
+            - Yes
+                - Remove label `status/triage`
+                - Add label `status/ready`
+                - Add appropriate `size/*` (if possible) and `type/*`
+            - No - needs clearer definition of done
+                - Remove label `status/triage`
+                - Add acceptance and label `status/incomplete`
+                - Add appropriate `size/*` (if possible) and `type/*`
+                - If activity is missing for 7 business days, follow up or close
+            - No - needs alignment
+                - Remove label `status/triage`
+                - Add label `status/discussion-needed`
+                - Add appropriate `size/*` (if possible) and `type/*`
+            - No - blocked by other work
+                - Remove label `status/triage`
+                - Add label `status/blocked`
+                - Add appropriate `size/*` (if possible) and `type/*`
+
+#### Pull Requests
```

Comment:
What problem is the PR workflow solving? I feel like the normal PR review cycle (requesting changes until code and behavior looks good and then approving and merging) already provides everything we need.

Reply:
This is attempting to solve two things:

1. Preventing PR from becoming stale via the same `status/triage` strategy we use for issues.
2. Encoding the idea that as maintainers we must take on two responsibilities. 1) approve the code 2) run through user acceptance.

That said, this was very quick thought process and as we continue to do more I can see that some adjustments can be made to simplify it. Will make a tweak here.